src/js/component.js

```diff
@@ -566,17 +566,46 @@ class Component {
   /**
    * Add an `event listener` to this `Component`s element.
    *
+   * ```js
```

Comment:
Yeah, I've rolled back the changes to this file. Looks like it was a rebase mixup.